### ISSUE TITLE
--watch flag for reads

### ISSUE THREAD

**Issue by edaniszewski** [vapor-ware/synse-cli#145]
Not a high priority right now, but an idea I had for the future is that there could be a `--watch` flag for reads, e.g. 
```
synse server read --watch=1s rack board device
```
where the value is interval duration to refresh the reading.

**Comment by edaniszewski** [comment]
instead of `--watch`, this could be `--stream`

depends on: https://github.com/vapor-ware/synse-client-go/issues/76

**Comment by edaniszewski** [comment]
closing in favor of #220
